pennylane/gradients/finite_difference.py

```diff
@@ -303,7 +308,9 @@ def _finite_diff_new(
         gradient_tapes.extend(g_tapes)
         shapes.append(len(g_tapes))
 
-    def processing_fn(results):
+    def _process_one_batch_result(tape, results, shapes, c0, f0, coeffs):

```

Comment:
Maybe it would be great here to specify that it is a batch of shots.

Reply:
Updated the docstring. :+1: